source/content/terminus/02-install.md

```diff
@@ -95,14 +95,33 @@ The commands below will:
 - Make the file executable
 - Add a symlink to your local `bin` directory for the Terminus executable
 
+<TabList>
+
+<Tab title="PHP 8.4+" id="php84" active={true}>
+
+```bash{promptUser: user}
+mkdir -p ~/terminus && cd ~/terminus
+curl -L https://github.com/pantheon-systems/terminus/releases/download/4.0.0/terminus.phar --output terminus
+chmod +x terminus
+./terminus self:update
+sudo ln -s ~/terminus/terminus /usr/local/bin/terminus
+```
+
+</Tab>
+
+<Tab title="PHP 8.2/8.3" id="php82-83">
```

Comment:
This looks confusing; maybe wrong titles? 8.2 and 8.3 are also supported by Terminus 4 (and it would be recommended). You should only go to 3.6.2 if you need 7.4-8.1

Reply:
Fair

```suggestion
<Tab title="PHP 8.3 and below" id="php83-">
```